util/reggen/access.py

```diff
@@ -79,6 +92,13 @@ class HwAccess(JsonEnum):
     'none': ("No Access Needed", HwAccess.NONE)
 }
 
+HWACCESS_RDL_MAP = {
```

Comment:
Please can we make this simpler so it's something like the following?
```python
HWACCESS_RDL_MAP = {
    HwAccess.HRO: AccessType.r,
    HwAccess.HRW: AccessType.rw,
    HwAccess.HWO: AccessType.w,
    HwAccess.NONE: AccessType.rw,
}
```
There's no real need for dictionaries as values, I don't think.

Also (as with `SWACCESS_RDL_MAP`) can this have a documentation comment?